### PR TITLE
Add further information on how to configure your deployment enviroment

### LINKED ISSUE #19: "Creating a stack hub" guide does not completely explain how to configure deployment environment
Looking at the "Configuring your deployment environment" section of the ["Creating a stack hub" guide](https://kabanero.io/guides/creating-a-stack-hub/#configuring-your-deployment-environment) there a few things that are hard to follow:

"You must update your Kubernetes operator custom resource (CR) definition file to reference your stack hub index file."   

1. Which CR? There are so many in Kubernetes. I am guessing this should be Kabanero instead of Kubernetes, but unclear as I have never executed these steps?

1. Where is that "Kubernetes operator custom resource (CR) definition file" located? Was it created in another guide (presumably installation) ? If so, a link to the guide would be helpful. As a reader, I would not know where to find that file. I know I could use `kubectl get <some type, name, or label> -o yaml` to get a copy of the existing file or `kubectl patch ...`, but I don't know what the selection criteria would be.

1. Once I make that edit, then what? Apply it back with `kubectl apply ...`? I am guessing that, at a minimum, yes, but unclear if anything else is needed, such as "waiting a few moments" or <restarting something>. Should there be a verification step, such as a kabanero CLI invocation or kicking off a new build using the new stack?

cc @SueChaplain , @groeges

### PR DESCRIPTION
**Pull request by SueChaplain**
Addressing #19, to improve how an architect/devops would configure their Kabanero
CRD to use customized application stacks.

Other improvements in this PR:

- Change title of guide to "Customizing application stacks", which is more accurate.
- Add further information for how developers would customize their local dev environments with links to other guides
- Fix ordered lists by adjusting indenting.

Closes: #19

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Review by groeges — APPROVED** [review]
Gives all the required info. LGTM.

**Comment by SueChaplain** [comment]
@nastacio - please can you review and confirm that these changes are appropriate to close issue #19

File: publish/working-with-stacks/working-with-stacks.md [hunk 1]
```diff
@@ -217,66 +217,94 @@ Now that your stack is available in your local repository, the next step is to r
...
 
-```
-stacks:
-  repositories:
-  - name: my-org-repository
-    https:
-      url: https://github.com/myorg/my-org-repository/releases/latest/download/my-org-repository-index.yaml
-```
+    ```
+    oc get kabanero <name> -o yaml > kabanero.yaml
```
Comment (nastacio) [inline publish/working-with-stacks/working-with-stacks.md]:
```
oc get kabanero <name> -n kabanero -o yaml > kabanero.yaml
```

**Review by nastacio — CHANGES_REQUESTED** [review]
I tested the modifications with a script created from copying-pasting instructions alone, so I know they will work for sure. There is only one missing bit: a namespace parameter in one of the `oc get` calls.

File: publish/working-with-stacks/working-with-stacks.md (hunk 1 above)
Comment (SueChaplain) [reply publish/working-with-stacks/working-with-stacks.md]: Fixed. Thank you! Could you please approve your review?

**Review by nastacio — APPROVED** [review]
